src/messages/messages.json

```diff
@@ -1,4 +1,17 @@
 {
   "run_failed": "Failed to parse the coverage XML report",
-  "exit_code": "The coverage XML report parsing finished with code "
+  "exit_code": "The coverage XML report parsing finished with code {0}",
+  "finding_coverage_report": "Finding Parasoft coverage XML report...",
+  "finding_coverage_report_in_working_directory": "Finding Parasoft SOAtest XML report in working directory {0}...",
+  "found_coverage_report": "Found Parasoft coverage XML report file: {0}",
+  "found_multiple_coverage_report": "Found {0} Parasoft coverage XML report files",
+  "coverage_report_not_found": "Parasoft coverage XML report not found at the specified location: {0}",
+  "converting_coverage_report_to_cobertura": "Converting Parasoft coverage report: {0} to cobertura report...",
+  "finding_java_in_java_or_parasoft_tool_install_dir": "Trying to find Java from Java or Parasoft tool installation directory: {0}...",
+  "using_java_to_convert_report": "Converting the report using Java...",
+  "java_or_parasoft_tool_install_dir_not_found": "Unable to process the XML report using Java because the Java or Parasoft tool installation directory is missing",
```

Comment:
Does this mean we’ve decided that if the Java environment is not specified, we will not attempt to use the JavaScript library to convert the report? Could you please clarify the reasoning behind this decision? (I don't mean that we should or shouldn’t do it, just curious about how the decision was made.)
Additionally, I may have some other comments about the messages after our discussion.